pkgs/development/compilers/ghc/common-hadrian.nix

```diff
@@ -355,7 +355,9 @@ let
     # documentation) makes the GHC RTS able to load static libraries, which may
     # be needed for TemplateHaskell. This solution was described in
     # https://www.tweag.io/blog/2020-09-30-bazel-static-haskell
-    lib.optionals enableRelocatedStaticLibs [
+    #
+    # Note `-fexternal-dynamic-refs` causes `undefined reference` errors when building GHC cross compiler for windows
+    lib.optionals (enableRelocatedStaticLibs && !targetPlatform.isWindows) [
```

Comment:
Taken from https://github.com/input-output-hk/haskell.nix/commit/16361e60723028f2e9c9839521ccab3f95f4a9cb#diff-46e7446cb5d647c0a0019e76d3a22acc891d1137e19ac7cd88c41fac6c6e39a5R316

I'm not sure what the x86_64 part there is about, should I cargo-cult?